### DISCUSSION TITLE
Help request: stack fails when 'CloudFrontOac' resource fails to create

### DISCUSSION THREAD

**Discussion by michael-mackenzie** [aws-solutions/workload-discovery-on-aws#455] [Q&A] [answered]
When attempting to deploy the solution, the stack fails to create on the error:

- Embedded stack arn:aws:cloudformation:ca-central-1:XXXXXXXXX:stack/workload-discovery-WebUiStack-YYYYYYYY/ZZZZZZZZZ was not successfully created: The following resource(s) failed to create: [CloudFrontOac].

Does anyone have any ideas as to why this may be happening?

Thanks!

**Comment by svozza** [comment, marked as answer]
This is probably because of a limitation in the number of characters in the OAC name field. We sue the stack name to ensure it is unique but if the WD stack name you chose is very long then it can mean the OAC name will be too long. Try deploying the solution with a shorter name.

**Comment by aws-ymson** [comment]
Closing this out: Workload Discovery on AWS has been deprecated as of 08/14/2026 and is no longer being maintained.
